20221202-035220
0.09971761703491211
898509
Traceback (most recent call last):
  File "prun_train/prune.py", line 250, in <module>
    main(exp,args)
  File "prun_train/prune.py", line 216, in main
    loose_model, prune_mask_dict, prune_bn_dict, mask_bn_channel_dict)
  File "prun_train/prune.py", line 175, in init_weights_from_loose_model
    prune_mask_dict[a[int(i)]].cpu().numpy())))
ValueError: invalid literal for int() with base 10: 'conv'
done
